### PR TITLE
Add AI/ML WG to the lifecycle documents.

### PR DESCRIPTION
**Pull request by mihaimaruseac**
We are already incubating, but we never finalize the lifecycle process. Filling the missing pieces with this PR.

### CODE REVIEW

**Comment by sevansdell** [comment]
> We are already incubating, but we never finalize the lifecycle process. Filling the missing pieces with this PR.
> 
> We are missing some documents in the WG repository, if that is a requirement before this can be merged I can work on having them created.

@mihaimaruseac TAC will look for these documents before approving lifecycle stage.

**Comment by mihaimaruseac** [comment]
Thank you for the context. I'll add them to the repo and come back here. Will mark this as draft until then

**Comment by mihaimaruseac** [comment]
This is now up to date, we have all the missing documents.

**Review by SecurityCRob — APPROVED** [review]

File: process/wg-lifecycle-documents/ai_ml_incubating_stage.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+### List WG Chairs
+
+The WG must have a minimum of 1 Chair
+
+* Mihai Maruseac, Google, mihaimaruseac
+* Jautau "Jay" White, Microsoft, camaleon2016
+
+### Working Group (WG) has met all Sandbox requirement
+
+* Applying directly to Incubating
```
Comment (sevansdell) [inline process/wg-lifecycle-documents/ai_ml_incubating_stage.md:12]: Add how the Sandbox requirements are completed:
Proposal of scope for review by TAC
This is to help ensure limited overlap with existing WGs
Have at least 3 interested individuals from at least 2 different organizations supporting the proposal (https://github.com/ossf/tac/blob/main/process/working-group-lifecycle.md)
1 TAC sponsor - Was it originally Dan A? Is it now Jay W?
TAC sponsor agrees to attend WG meetings regularly
TAC sponsor does not need to have a formal role in WG, e.g., chair
TAC sponsor requests TAC approval

File: process/wg-lifecycle-documents/ai_ml_incubating_stage.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+The WG must have a charter or mission statement for review by TAC
+
+* https://github.com/ossf/ai-ml-security/blob/main/mvsr.md
+
+### Governance
+
+WG must have documented, initial group governance.
+
+* https://github.com/ossf/ai-ml-security/blob/main/doc/CHARTER.md
```
Comment (sevansdell) [inline process/wg-lifecycle-documents/ai_ml_incubating_stage.md:42]: If you update the MVSR link above, the text on this charter should be updated as well.

File: process/wg-lifecycle-documents/ai_ml_incubating_stage.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+WG must have met publicly at least 5 times in the last quarter since becoming
+Sandbox
+
+* 2024: https://docs.google.com/document/d/1YNP-XJ9jpTjM6ekKOBgHH8-avAws2DVKeCpn858siiQ/edit
+* 2023: https://docs.google.com/document/d/1hEJdr7yXv5tfYpUfn_e7DAUTyvoU6mY0b9eH6EXAHaI/edit
+
+WG must have defined Contributor Guide
+
+* https://github.com/ossf/ai-ml-security?tab=readme-ov-file#how-to-participate
```
Comment (sevansdell) [inline process/wg-lifecycle-documents/ai_ml_incubating_stage.md:52]: Looks like the "how to participate" has list of current interlocks. Recommend moving this to MVSR, and linking to MVSR in the participation guide.

**Review by sevansdell — CHANGES_REQUESTED** [review]
I'm highly supportive of this incubating lifecycle status. To prepare for continued growth potential for this WG, I put in a few comments to crisp up the linked AIML WG documents. Am happy to help do this if needed (wearing my AIML WG participant hat).

**Comment by lehors** [comment]
This PR should include changing the TAC README to add a link from the "Incubating" status corresponding to the AI/ML WG to the lifecycle doc being submitted.
Thanks.

File: process/wg-lifecycle-documents/ai_ml_incubating_stage.md (hunk 1 above)
Comment (mihaimaruseac) [reply process/wg-lifecycle-documents/ai_ml_incubating_stage.md]: Thank you! Included these ones and then I'll work on the docs on the WG side.

**Comment by mihaimaruseac** [comment]
Added the "Incubating" status summary, but now will need to work on documents on the WG side. Will mark this as draft.

**Comment by SecurityCRob** [comment]
We're still waiting on requested changes so that this PR can be merged.

**Comment by sevansdell** [comment]
Any update here? Would love to get the AIML WG set as incubating.

File: process/wg-lifecycle-documents/ai_ml_incubating_stage.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+* Sarah Evans, Dell Technologies
+* Yotam Perkal, Rezilion
+
+and many others
+
+### Mission of the Working Group
+
+The WG must have a charter or mission statement for review by TAC
+
+* https://github.com/ossf/ai-ml-security/blob/main/mvsr.md
```
[the remark this answers is not in the document]
Comment (mihaimaruseac) [reply process/wg-lifecycle-documents/ai_ml_incubating_stage.md:40]: Updating this in https://github.com/ossf/ai-ml-security/pull/24

File: process/wg-lifecycle-documents/ai_ml_incubating_stage.md (hunk 3 above)
Comment (mihaimaruseac) [reply process/wg-lifecycle-documents/ai_ml_incubating_stage.md:56]: Moving them to an updated MVSR in https://github.com/ossf/ai-ml-security/pull/24

**Comment by mihaimaruseac** [comment]
Apologies for the delay, working on this right now and hope to have it done by end of today

File: process/wg-lifecycle-documents/ai_ml_incubating_stage.md (hunk 2 above)
Comment (mihaimaruseac) [reply process/wg-lifecycle-documents/ai_ml_incubating_stage.md:46]: Updating it in https://github.com/ossf/ai-ml-security/pull/24

**Comment by mihaimaruseac** [comment]
I think this is now ready for review. Thank you for the previous rounds of comments!

File: process/wg-lifecycle-documents/ai_ml_incubating_stage.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+WG must have met publicly at least 5 times in the last quarter since becoming
+Sandbox
+
+* 2024: https://docs.google.com/document/d/1YNP-XJ9jpTjM6ekKOBgHH8-avAws2DVKeCpn858siiQ/edit
+* 2023: https://docs.google.com/document/d/1hEJdr7yXv5tfYpUfn_e7DAUTyvoU6mY0b9eH6EXAHaI/edit
+
+WG must have defined Contributor Guide
+
+* https://github.com/ossf/ai-ml-security/blob/main/README.md#how-to-participate
```
Comment (marcelamelara) [inline process/wg-lifecycle-documents/ai_ml_incubating_stage.md:57]: Just a really minor nit: The Slack channel linked in the README still points to the old name that used underscores. You may want to update that so it points to the updated `#wg-ai-ml-security` channel.

**Review by marcelamelara — APPROVED** [review]
Thanks @mihaimaruseac ! I absolutely support this application, looking forward to continuing to grow the community!

File: process/wg-lifecycle-documents/ai_ml_incubating_stage.md (hunk 5 above)
Comment (mihaimaruseac) [reply process/wg-lifecycle-documents/ai_ml_incubating_stage.md:57]: Thank you for catching that! I updated the repo's README.

**Review by mlieberman85 — APPROVED** [review]
LGTM!

**Review by camaleon2016 — APPROVED** [review]

**Review by lehors — CHANGES_REQUESTED** [review]
This looks good but it is still missing the update to the README:

> This PR should include changing the TAC README to add a link from the "Incubating" status corresponding to the AI/ML WG to the lifecycle doc being submitted.
> Thanks.

**Review by torgo — APPROVED** [review]
lgtm

**Review by bobcallaway — APPROVED** [review]

**Comment by mihaimaruseac** [comment]
> This looks good but it is still missing the update to the README:
> 
> > This PR should include changing the TAC README to add a link from the "Incubating" status corresponding to the AI/ML WG to the lifecycle doc being submitted.
> > Thanks.

Oh, sorry, I missed this before. This should now be updated.

**Review by lehors — APPROVED** [review]
LGTM. Thanks.

**Review by steiza — APPROVED** [review]

**Pull request merged**